src/npc.h

```diff
@@ -168,7 +168,7 @@ class job_data
             const std::pair<activity_id, int> &b ) {
                 return a.second > b.second;
             } );
-            for( std::pair<activity_id, int> elem : pairs ) {
+            for( const std::pair<activity_id, int> &elem : pairs ) {
```

Comment:
I suspect clang10 will complain about this
```suggestion
            for( const std::pair<const activity_id, int> &elem : pairs ) {
```